governance-exercises/DAO Masons poll-exercise-04.md

```diff
@@ -0,0 +1,58 @@
+
+Since the fixes will be ready in 8 hours, I would set the poll to 7 hours as the poll shouldn't slow down the emergency response. I'll still coordinate an emergency governance call to discuss solutions.
+
+I'd work to get the poll live asap, and hopefully be able to delegate to someone on my team to begin reaching out to every known stakeholder. We see that the governance/delegates folder in the community repo has handles for members. We'd spend the rest of the time before the poll ends to reach out to everybody we can and alert them that there is an emergency vote in progress.
+
+### Draft any forum responses required or implied by your course of action.
+
+As Governance Facilitator I am processing a request from Protocol Facilitator Imogen to declare this an Urgent or Emergency situation and will refer to [MIP24 Emergency Response](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) as appropriate.
+
+I'm taking the following actions:
+
+1. Drafting an emergency poll
+2. Coordinating an emergency governance call to discuss solutions and a plan for subsequent actions
+
+I have some clarifying questions for Imogen and Geneveive and will ask them publicly to help inform the voters in the poll.
+
+1. MKR holders will want to understand the risk to their funds. Please expand on the statement "Fix A is likely to incur losses to MKR holders of approximately 20 million DAI over two weeks."
+
+   - Would this loss be proportional across all MKR holders according to how much MKR they hold?
+   - Would it be represented as a drop in MKR trading price which could be recoverable?
+
+2. Rather than a holding a poll, would it be feasible to put Fix A and Fix B up concurrently as separate executive votes, and let MKR holders decide on Fix A by voting to pass it or not?
+
+3. When will the fixes be ready. You said "very shortly." How long will that be?
+
+A note to the community - please keep public speculation to a minimum as broadcasting this vulnerability increases the likelihood it will be discovered before it is patched.
+
+### Draft any polls required or implied by your course of action.
+
+See accompanying file in PR
+
+### Create a draft PR to this repository containing your response materials.
+
+### Notes
+
+We want to note that we found the [Emergency Response](https://mips.makerdao.com/mips/details/MIP24#MIP24c2) documents to be disappointing in its guideance. One example: it refers to the urgent response procedure outlined in MIP24c4 and then MIP24c4 doesn't outline a procedure. Lots of room for improvement here. We were hoping for a checklist to guide us in an emergency and we ended up spending hours trying to discern what we should do. I guess this exercise prepares us somewhat, and if we get the role we'll want to work with AVCs to improve these docs.
```

Comment:
Yes. This is what happens when major stakeholders half-ass edit important documents and no one holds them to account. 

The original was slightly better, though I agree it could have been improved.

Reply:
MIPs are no longer really a thing, in practice you would need to add some emergency response guidelines to the scopes.